lib/core/boot/boot.ts

```diff
@@ -61,6 +65,33 @@ export const bootstrap = (): FluxifyServer => {
 
 			if (targetRoute) {
 				try {
+					if (useThrottle) {
+						const entry = global.server.throttle[request.ip]?.[endpoint];
+						if (entry) {
+							if (entry.exp < Date.now()) {
+								entry.exp = Date.now() + config.throttleTtl * 1000;
+								entry.hits = 0;
+							}
+							entry.hits += 1;
```

Comment:
Are you sure u want to add before checking, because later u add after checking if any entry exists.